src/Host/Client/Test/Mocks/RSessionMock.cs

```diff
@@ -31,7 +31,7 @@ public sealed class RSessionMock : IRSession {
 
         public Task<IRSessionEvaluation> BeginEvaluationAsync(CancellationToken cancellationToken = default(CancellationToken)) {
             Evaluation = new RSessionEvaluationMock();
-            BeforeRequest?.Invoke(this, new RRequestEventArgs(Evaluation.Contexts, Prompt, 4096, true));
+            BeforeRequest?.Invoke(this, new RBeforeRequestEventArgs(Evaluation.Contexts, Prompt, 4096, true));
```

Comment:
Done

---
In reply to: [71067946](https://github.com/Microsoft/RTVS/pull/2080#discussion_r71067946) [](ancestors = 71067946)